heron/instance/src/java/BUILD

```diff
@@ -13,6 +13,7 @@ instance_deps_files =  \
         "//heron/common/src/java:config-java",
         "//heron/common/src/java:network-java",
         "//heron/common/src/java:utils-java",
+        "@commons_cli_commons_cli//jar"
```

Comment:
If we are introducing this dep, we need to add a shading rule for this.

Reply:
I have added a rule to shade this dep